apps/governance-e2e/src/integration/view/validators.cy.ts

```diff
@@ -37,206 +37,211 @@ context('Validators Page - verify elements on page', function () {
   });
 
   describe('with wallets disconnected', { tags: '@smoke' }, function () {
-    describe('description section', function () {
-      it('Should have validators tab highlighted', function () {
-        verifyTabHighlighted(navigation.validators);
-      });
-
-      it('Should have validators ON VEGA header visible', function () {
-        verifyPageHeader('Validators');
-      });
+    it('Should have validators tab highlighted', function () {
+      verifyTabHighlighted(navigation.validators);
+    });
 
-      it('Should have Staking Guide link visible', function () {
-        // 2001-STKE-003
-        cy.get(guideLink)
-          .should('be.visible')
-          .and('have.text', 'Read more about staking on Vega')
-          .and(
-            'have.attr',
-            'href',
-            'https://docs.vega.xyz/mainnet/concepts/vega-chain/#staking-on-vega'
-          );
-      });
+    it('Should have validators ON VEGA header visible', function () {
+      verifyPageHeader('Validators');
     });
-    describe(
-      'Should be able to see validator list from the staking page',
-      { tags: '@regression' },
-      function () {
-        // 2001-STKE-050
-        it('Should be able to see validator names', function () {
-          cy.get('[col-id="validator"] > div > span')
-            .should('have.length.at.least', 1)
-            .each(($name) => {
-              cy.wrap($name).should('not.be.empty');
-            });
-        });
 
-        it('Should be able to see validator stake', function () {
-          cy.getByTestId('total-stake')
-            .should('have.length.at.least', 1)
-            .each(($stake) => {
-              cy.wrap($stake).should('not.be.empty');
-            });
-        });
+    it('Should have Staking Guide link visible', function () {
+      // 1002-STKE-003
+      cy.get(guideLink)
+        .should('be.visible')
+        .and('have.text', 'Read more about staking on Vega')
+        .and(
+          'have.attr',
+          'href',
+          'https://docs.vega.xyz/mainnet/concepts/vega-chain/#staking-on-vega'
```

Comment:
Question (non-blocking): I wonder if we could derive this value rather than hardcoding it, that way in case it changes the tests won't fail. In the apps we have an function that allows this, are you able to make use of it?

https://github.com/vegaprotocol/frontend-monorepo/blob/caaa82f60fe5c4b70da912ee3bf34ccfed8bc667/libs/environment/src/hooks/use-links.ts#L79